src/ResourceManager/Resources/Commands.Resources/help/AzureRM.ManagementGroups.md

```diff
@@ -0,0 +1,31 @@
+---
+Module Name: AzureRM.ManagementGroups
+Module Guid: {{ Update Module Guid }}
```

Comment:
Please fill out all of these fields (module guid should be in psd1, download link will be https://docs.microsoft.com/en-us/powershell/module/azurerm.managementgroups, help version can be 1.0.0, and locale is en-US).

Reply:
Thanks. Done.